src/lib/components/app-sidebar.svelte

```
@@ -211,18 +166,31 @@
 	</Sidebar.Content>
-	<Sidebar.Footer class="border-t-primary">
+	<Sidebar.Footer>
 		<Sidebar.Menu>
-			<Sidebar.MenuItem>
+			<Sidebar.MenuItem class="m-1">
+				<Sidebar.MenuButton
+					isActive={currentitem == 'Settings'}
+					onclick={() => handleMenuClick('Settings')}
+				>
+					{#snippet child({ props })}
+						<a href="/organization" {...props}>
```

ESLint

[error] Unexpected href link without resolve().

(svelte/no-navigation-without-resolve)